### PR TITLE
issue204: application-data.json file created for test-suite run

### PR DESCRIPTION
**Pull request by sunil-kumar-meena**
fixes https://github.com/openBackhaul/AdministratorAdministration/issues/204

reviewer is @PrathibaJee

### CODE REVIEW

File: server/application-data/application-data.json [hunk 1]
```diff
@@ -1,19 +1,67 @@
...
+      "administrator-name": "administrator",
+      "user-name": "admin.user",
+      "authorization": " *** Basic Ath Code *** ",
+      "allowed-methods": "DELETE"
+    },
+    {
+      "administrator-id": "3",
+      "administrator-name": "administrator",
+      "user-name": "admin.user",
+      "authorization": "Basic aXN3YXJ5YWEuc3ViYXNoY2hhbmRyYW46aXN3YXJ5YWE=",
```
Comment (IswaryaaS) [inline server/application-data/application-data.json:21]: please remove the valid authorization code in github as it is confidential information

**Comment by sunil-kumar-meena** [comment]
> please remove the valid authorization code in github as it is confidential information

thank you for the comment, it's removed

File: server/application-data/application-data.json [hunk 2]
```diff
@@ -1,19 +1,67 @@
...
-        "administrator-id" : "2",
-        "administrator-name" : "administrator",
-        "user-name" : "admin.user",
-        "authorization" : " *** Basic Ath Code *** ",
-        "allowed-methods" : "PUT"
-      }
-    ]
-  }
\ No newline at end of file
+  "administrator-credential-list": [
```
Comment (IswaryaaS) [inline server/application-data/application-data.json:2]: i believe we would need a admin user having "ALL" permission. Could you please add one for such user. We can have it in id "1"

Comment (sunil-kumar-meena) [reply server/application-data/application-data.json:2]: done

**Review by IswaryaaS — APPROVED** [review]

**Pull request merged**